templates/triton_services/pytorch_app.py

```diff
@@ -0,0 +1,56 @@
+"""
+Some of those code are taken from Hugging Face's documentation and modified to run on
+Ray Serve.
+
+For Hugging Face's original doc see:
+    https://huggingface.co/docs/diffusers/en/optimization/torch2.0#torchcompile
+"""
+
+from io import BytesIO
+from fastapi import FastAPI
+import torch
+
+from ray import serve
+from diffusers import DiffusionPipeline
+from fastapi.responses import StreamingResponse
+
+app = FastAPI()
+
+
+@serve.deployment(
+    ray_actor_options={
+        "num_gpus": 1,
```

Comment:
not needed; default

Reply:
Just double check this is talking about "num_cpus" being default and not needed right? "num_gpus" is still needed just so the deployment can be scheduled onto GPU nodes.

Reply:
yes